examples/simple/universal/z_simple.cxx

```diff
@@ -30,24 +30,26 @@ class CustomerClass {
     CustomerClass& operator=(const CustomerClass&) = delete;
     CustomerClass& operator=(CustomerClass&&) = delete;
 
-    CustomerClass() : session(nullptr), pub(nullptr) {
+    CustomerClass(const KeyExprView& keyexpr) : session(nullptr), pub(nullptr) {
         Config config;
         Session s = std::get<Session>(open(std::move(config)));
```

Comment:
Not really the PR's goal, but could we avoid having examples where we encourage users to construct `Session`s in classes? It's literally the worst thing they could do.

Reply:
Will do it in separate PR

Reply:
https://github.com/eclipse-zenoh/zenoh-cpp/issues/51